manifests/m/Microsoft/XNARedist/4.0.30901.0/Microsoft.XNARedist.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.XNARedist
+PackageVersion: 4.0.30901.0
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft XNA Framework Redistributable 4.0 Refresh

```

Comment:
```suggestion
PackageName: Microsoft XNA Framework Redistributable Refresh
```

Reply:
Thought PackageName needed to match what was shown in the Add and Remove Programs list

Reply:
![image](https://user-images.githubusercontent.com/1053504/132876107-1d648eb5-a080-4ab7-a987-12eed84f810e.png)

Reply:
without the version number included for now. It'll be fixed up later in 1.1.

Reply:
aah ok got it